Kickstarter-iOS/Views/Controllers/ProfileSavedProjectsViewController.swift

```diff
@@ -0,0 +1,67 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+internal protocol ProfileSavedProjectsViewControllerDelegate: class {
```

Comment:
At all useful to be DRY and use the same protocol and perhaps pass in the controller?